README.md

```diff
@@ -51,14 +52,14 @@ Default domain is [http://dev.typo3.vm]()
 Build and run an image with installed and fully functional introduction package:
 
 ```
-docker-compose -f docker-compose.yml build
-docker-compose -f docker-compose.yml -f docker-compose.demo.yml up -d
+docker-compose build
+docker-compose -f docker-compose.yml -f docker-compose.demo.yml up --build -d
 ```
 
 Use the following command to install the introduction package using TYPO3 console:
 
 ```
-docker-compose exec /bin/bash ./install.sh
+docker-compose exec webserver /bin/bash ./install.sh
```

Comment:
This was missing the service name. Without it docker-compose doesn't know in which container to execute the command